source/tools/index.rst

```diff
@@ -10,4 +10,5 @@ Tools
    StitchTool
    GenContainer
    ConfigTools
-   GpioTool
\ No newline at end of file
+   GpioTool
+   UcodeUtility
```

Comment:
Add a tutorial section - Mentioned about the tools to corrupt image at static and at run time.